src/Aggregation.ts

```diff
@@ -0,0 +1,54 @@
+/** AggreationType will be used to create type of aggreation when querying history states.
+  <ul>
+  <li>AggregationType.COUNT: count function.</li>
+  <li>AggregationType.SUM: sum function. </li>
+  <li>AggregationType.MAX: max function.</li>
+  <li>AggregationType.MIN: min function.</li>
+  <li>AggregationType.MEAN: mean function.</li>
+  </ul>
+*/
+export const FunctionType = {
+    COUNT: "COUNT",
+    SUM: "SUM",
```

Comment:
@shinsuke-nara The string literal is good for pure typescript project. However, we need to consider that our developers may not use typescript in their project. 

Type script compiler doesn't generate js code for string literal. You can try the following code in typescript [playground](https://www.typescriptlang.org/play/index.html
): 

```js
type FunctionType = "SUM" | "COUNT" | "MIN";

function testFunc(ft: FunctionType) {
    alert(ft);
}

testFunc("SUM")

```

I think it is better to use const, which can expose the values to developers.